doc/Section_start.txt

```diff
@@ -379,7 +379,7 @@ stitch-related options:
 "set stitch"_set.html
 "reset_time"_reset_time.html :ul
 
-See the am_path, potts_additive, and stitch sub-directories in the
+See the am_path and stitch sub-directories in the
```

Comment:
Don't you wan to add the am_ellipsoid directory to these two?
Those input scripts use stitch as well.

Reply:
Well -- this refers to 'stitch' specifically; the am_path and stitch directories have examples specifically targeting stitch.  
Which reminds -- we probably need to look through the examples in those directories and make sure that they don't call out potts/additive.

Reply:
The examples/am_ellipsoid input scripts use the dump stitich command which also uses the Stitich library, correct ?
Dump stitch is one of the 3 commands listed above this line on the doc page.